docs/DataWorkflows/FetchingYourData.md

```diff
@@ -246,6 +243,212 @@ Critically, the reference to the EXACT version of the file you used in your
+
+Now that we figured out what files are available for a given participant, let’s “drop” the zipped participant data. This doesn’t do anything to the original data on PMACS, but just removes the downloaded data in the cloned dataset in your project directory, which saves space.
+
+```bash
+# drop the file
+datalad drop --nocheck $file
+```
+
+This is what the output should look like: 
+
+```bash
+It is highly recommended to configure Git before using DataLad. Set both 'user.name' and 'user.email' configuration variables.
+drop(ok): sub-NevilleLongbottom_fmriprep-20.2.3.zip (file)
+```
+
+Now that we know how to look for available data in a zipped static PMACS dataset, we’re ready to scale things up and download all required data for the whole dataset!
+
+Here's how this may look unzipping on a larger scale: 
+
+
+
+```bash
+#!/bin/bash
+# conda activate dlad
+
+# clone static data of interest from PMACS, here LINC_PNC#~FMRIPREP_zipped, 
+# but can be replaced with anything in the "Clone URL" column here https://pennlinc.github.io/docs/DataTasks/AvailableStaticData/
+datalad clone ria+ssh://dumbledore@bblsub.pmacs.upenn.edu:/static/LINC_PNC#~FMRIPREP_zipped /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/datalad
+
+# do this so that you don't have to enter your pmacs password 1 million times when datalad getting files
+# eval $(ssh-agent)
+# ssh-add ~/.ssh/id_rsa_pmacs
+
+# The clone files exist only remotely until we get them locally, so once we have a clone of the data, we need to "datalad get" the files we are interested in. 
+cd /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/datalad
+
+# to see the history of what happened to a zipped dataset (i.e. get simlinks without actually getting the data locally)
+# datalad get -n .
+
+# ----------------------------------------
+## get a few subjects for testing ###
+# ----------------------------------------
+
+subject_ids=("HarryPotter","LordVoldemort","NevilleLongbottom")
+
+for id in "${subject_ids[@]}"; do
+	# get filename
+	file="sub-${id}*zip"
+
+	# make directory to store unzipped files
+	mkdir -p /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/zipped
+	mkdir -p /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/unzipped
+
+	# get the file
+	datalad get $file -J 8
+
+	# copy the zip to another directory outside of the official datalad clone so that I'm not messing with it
+	cp $file /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/zipped
+
+	# unzip the files outside of the datalad clone
+	unzip "/cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/zipped/$file" -d /cbica/projects/hogwarts/data/PNC/PNC_fMRIPrep_v20_2_3/unzipped/
+
+	# # drop the file
+	datalad drop --nocheck $file
+
+done
+
+# -------------------------------------------------
+#### get zip files for the entire PNC dataset #####
+# -------------------------------------------------
+
+# Note: if this requires inputting the password every time, may need to switch to python and use glob patterns
```

Comment:
can remove this line, not needed here

Note: if this requires inputting the password every time, may need to switch to python and use glob patterns